helm/abm/templates/_helpers.tpl

```diff
@@ -30,3 +30,25 @@ Create chart name and version as used by the chart label.
 {{- define "airbyte-module.chart" -}}
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
+{{/*
+returns true if the connection format is equal to the format in the
+additional properties defined for a connector. If the additional properties does not
+contain a format key then they are considered equal.
+*/}}
+{{- define "airbyte-module.is-format-equal" -}}
+{{- if hasKey .additionalProperties "expected_format" -}}
+{{- if eq (get .additionalProperties "expected_format") .connectionFormat -}}
+{{- true }}
+{{- end -}}
+{{- else -}}
+{{- true }}
```

Comment:
true in both cases?

Reply:
Yes, if the key is not present or its present and equal.